packages/db/src/async_operations.ts

```diff
@@ -0,0 +1,33 @@
+import { toSnake } from 'snake-camel';
+
+interface CamelCaseAsyncOperation {
```

Comment:
I think this should be named DynamoAsyncOperation or something like

Reply:
Same comment as the RDS suggestion.